### PR TITLE
[BUGFIX] is numeric check of id at product save

### PR DESCRIPTION
**Pull request by xandrej**
__Tasks:__
- [x] done
- [ ] todo
- [ ] got feedback

__Informations: product save bugfix

| Q            | A
| ------------- | ---
| Bug fix?      | yes
| New feature?  | no
| Tests pass?  | yes
| Fixed tickets | https://github.com/massiveart/POOL-ALPIN/issues/1277
| Doc          | none

### CODE REVIEW

**Comment by michaelzangerle** [comment]
choose a meaningful PR title and use [FEATURE]/[BUGFIX]/...

File: Resources/public/js/components/products/components/detail-form/main.js [hunk 1]
```diff
@@ -97,7 +97,7 @@ define([
                 var data = this.sandbox.form.getData(formSelector),
                     supplierId;
 
-                if (data.id === '') {
+                if (!$.isNumeric(data.id)) {
```
Comment (michaelzangerle) [inline Resources/public/js/components/products/components/detail-form/main.js]: use the jquery abstraction

Comment (michaelzangerle) [inline Resources/public/js/components/products/components/detail-form/main.js]: an id could also be a hash

**Comment by turbo-ele** [comment]
unfortunately this does not solve the real issue and is just a quick fix, since the real problem is that the entity seems to return strange id's.

ACK anyway

**Pull request merged**